src/src/PluginStructs/P036_data_struct.cpp

```diff
@@ -30,25 +31,43 @@ void P036_data_struct::reset() {
   }
 }
 
+# ifdef P036_FONT_CALC_LOG
+String tFontSettings::FontName() const {
+  if (fontData == ArialMT_Plain_24)
+    return "Arial_24";
```

Comment:
You could have that method return a `const __FlashStringHelper *` instead of s `String`, but should at least wrap the literal strings in the regular `F("string")` (flash-string) macro, to move the string from code segment to flash.

Reply:
Done!